docs/_pages/vbrmethods/viscous.md

```diff
@@ -8,6 +8,7 @@ The available viscous methods are:
 * `HK2003` [documentation](/vbr/vbrmethods/visc/hk2003/): Steady state olivine flow law from from Hirth and Kohlstedt 2003
 * `HZK2011` [documentation](/vbr/vbrmethods/visc/hzk2011/): Steady state olivine flow law from Hansen et al., 2011
 * `xfit_premelt` [documentation](/vbr/vbrmethods/visc/xfitpremelt/): Steady state flow law for pre-melting viscosity drop, Yamauchi and Takei, 2016.
+* `BKHK2023` [documentation](/vbr/vbrmethods/visc/bkhk2023/): Description here!
```

Comment:
this needs a one-line summary

Reply:
Feel free to change the wording, but here's a version that's sufficient from my perspective:

```suggestion
* `BKHK2023` [documentation](/vbr/vbrmethods/visc/bkhk2023/): steady state solution for the dislocation backstress model of Breithaupt et al., 2023.
```